src/mpi/romio/test/runtests.in

```diff
@@ -22,7 +22,8 @@ MAKE="@MAKE@"
 srcdir=@srcdir@
 check_canrun=0
 subset_only=0
-FILENAME=test
+PREFIX_FILENAME=
+FILENAME=
```

Comment:
Could you explain the fix?

Reply:
PREFIX_FILENAME will be `romio_test_` if not set by the users.
FILENAME will be `PREFIX_FILENAME`_`executable_name`
FILENAME is used at the mpirun command line later.
This should make the output files unique to each mpirun, in case
running make check in parallel.